.gitignore

```diff
@@ -0,0 +1,6 @@
+build/
+meta-openembedded
+meta-rauc
+meta-rauc-community
+meta-virtualization
+poky
```

Comment:
Please, add a comment, why you exclude this folders (build folder is understandable) in order everybody to know

Reply:
Makes sense, thanks